bin/pycbc_aligned_bank_cat

```diff
@@ -170,9 +171,27 @@ if options.metadata_file:
                                      contenthandler=LIGOLWContentHandler)
 else:
     outdoc = None
-tmpltbank.output_sngl_inspiral_table(options.output_file, temp_bank,
-    metricParams, ethincaParams, programName=__program__,
-    optDict=options.__dict__,
-    outdoc=outdoc, comment="", version=pycbc.version.git_hash,
-    cvs_repository='pycbc/'+pycbc.version.git_branch,
-    cvs_entry_time=pycbc.version.date)
+if options.output_file.endswith(('.xml','.xml.gz','.xmlgz')):
+    tmpltbank.output_sngl_inspiral_table(options.output_file, temp_bank,
+        metricParams, ethincaParams, programName=__program__,
+        optDict=options.__dict__,
+        outdoc=outdoc, comment="", version=pycbc.version.git_hash,
+        cvs_repository='pycbc/'+pycbc.version.git_branch,
+        cvs_entry_time=pycbc.version.date)
+elif options.output_file.endswith(('.h5','.hdf','.hdf5')):
+    out_fp = h5py.File(options.output_file, 'w')
+    out_fp['mass1'] = temp_bank[:,0]
+    out_fp['mass2'] = temp_bank[:,1]
+    out_fp['spin1z'] = temp_bank[:,2]
+    out_fp['spin2z'] = temp_bank[:,3]
+    out_fp['f_lower'] = [options.f_low] * len(temp_bank[:,0])
+    tmplt_durations = []
+    for i in xrange(len(temp_bank[:,0])):
+        gwflit = get_waveform_filter_length_in_time
+        wfrm_length = gwflit('SPAtmplt', mass1=temp_bank[i,0],
+                             mass2=temp_bank[i,1], f_lower=options.f_low,
+                             phase_order=7)
+        tmplt_durations.append(wfrm_length)
+    out_fp['template_duration'] = tmplt_durations
+    out_fp['approximant'] = ['TaylorF2'] * len(temp_bank[:,0])
+    out_fp.close()
```

Comment:
I'm adding a raise ValueError condition in case this happens.

However this is temporary code until @cdcapano adds support for this in the Bank class.